Packs/Slack/Integrations/SlackV3/SlackV3.yml

```diff
@@ -1,31 +1,23 @@
-sectionOrder:
-- Connect
-- Collect
-category: Messaging and Conferencing
```

Comment:
@maimorag I'm not sure if this was intentional, but it looks like all the section ordering for XSIAM was removed. I'd venture a guess and say demisto-sdk format did a number on the yml.